portfolio/src/main/webapp/hobbies.html

```diff
@@ -0,0 +1,66 @@
+            <span class="navbar-toggler-icon"></span>
+          </button>
+          <div class="collapse navbar-collapse" id="navbarNavAltMarkup">
+            <div class="navbar-nav">
+            <a class="nav-item nav-link active" href="index.html">Home <span class="sr-only">(current)</span></a>
+            <a class="nav-item nav-link active" href="#">Projects</a>
+            <a class="nav-item nav-link active" href="#">Hobbies</a>
+            </div>
+          </div>
+        </nav>
+      </div>
+      <div id="content">
+        <div id="title">
+          <h1>My hobbies</h1>
+        </div>
+      </div>
+      <div class="container">
+        <div class="row justify-content-center">
+          <div class="col-12 col-sm-6 col-md-4">
+            <div class="card">
+              <img src="images/intramuros.jpg" class="card-img-top" alt="Photo of my team after winning the state championship.">
+              <div class="card-body">
+                <h5 class="card-title">Playing Basketball</h5>
+                <p class="card-text">I played varsity basketball for three years. We even won a state championship on my senior year!</p>
+              </div>
+            </div>
+          </div>
+          <div class="col-12 col-sm-6 col-md-4">
+            <div class="card">
+              <img src="images/book.jpg" class="card-img-top" alt="Photo of a book with a forest-like background.">
+              <div class="card-body">
+                <h5 class="card-title">Reading</h5>
+                <p class="card-text">I read mostly science fiction warfare and some self development books. My favorit saga: Percy Jackson!</p>
+              </div>
+            </div>
+          </div>
+          <div class="col-12 col-sm-6 col-md-4">
+            <div class="card">
+              <img src="images/cartoon.jpg" class="card-img-top" alt="Photo of the Yu Gi Oh necklace made of silver.">
+              <div class="card-body">
+                <h5 class="card-title">Watching Cartoons</h5>
+                <p class="card-text">Big fan of some cartoons. Especially: Yu Gi Oh, Dragon Ball Z, Naruto and Avatar: The leyend of Ang.</p>
+              </div>
+            </div>
+          </div>    
+        </div>
+      </div>
+      <script src="https://code.jquery.com/jquery-3.5.1.slim.min.js" integrity="sha384-DfXdz2htPH0lsSSs5nCTpuj/zy4C+OGpamoFVy38MVBnE+IbbVYUew+OrCXaRkfj" crossorigin="anonymous"></script>
+      <script src="https://cdn.jsdelivr.net/npm/popper.js@1.16.0/dist/umd/popper.min.js" integrity="sha384-Q6E9RHvbIyZFJoft+2mJbHaEWldlvI9IOYy5n3zV9zzTtmI3UksdQRVvoxMfooAo" crossorigin="anonymous"></script>
+      <script src="https://stackpath.bootstrapcdn.com/bootstrap/4.5.0/js/bootstrap.min.js" integrity="sha384-OgVRvuATP1z7JjHLkuOU7Xw704+h835Lr+6QL9UvYjZE3Ipu6Tp75j7Bh/kR0JKI" crossorigin="anonymous"></script>
```

Comment:
You'll want to include these scripts up in the <head> section above:

```suggestion
<head>
  <meta charset="UTF-8">
  <title>My Hobbies</title>
  <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.5.0/css/bootstrap.min.css" integrity="sha384-9aIt2nRpC12Uk9gS9baDl411NQApFmC26EwAOH8WgZl5MYYxFfc+NcPb1dKGj7Sk" crossorigin="anonymous">
  <link rel="stylesheet" href="style.css">
  <script src="script.js"></script>
  <script src="https://code.jquery.com/jquery-3.5.1.slim.min.js" integrity="sha384-DfXdz2htPH0lsSSs5nCTpuj/zy4C+OGpamoFVy38MVBnE+IbbVYUew+OrCXaRkfj" crossorigin="anonymous"></script>
  <script src="https://cdn.jsdelivr.net/npm/popper.js@1.16.0/dist/umd/popper.min.js" integrity="sha384-Q6E9RHvbIyZFJoft+2mJbHaEWldlvI9IOYy5n3zV9zzTtmI3UksdQRVvoxMfooAo" crossorigin="anonymous"></script>
  <script src="https://stackpath.bootstrapcdn.com/bootstrap/4.5.0/js/bootstrap.min.js" integrity="sha384-OgVRvuATP1z7JjHLkuOU7Xw704+h835Lr+6QL9UvYjZE3Ipu6Tp75j7Bh/kR0JKI" crossorigin="anonymous"></script>
</head>
```

Reply:
Ack! Thing is that the bootstap documentation recommends to including those files at the end of the file so that the front-end may load first and then the JS files. Still made the change!